lib/travis/build/script/go.rb

```diff
@@ -110,10 +110,10 @@ def uses_make
           
           # see https://golang.org/doc/go1.6#go_command
           def uses_15_vendoring
```

Comment:
Oh, thanks! I'm, uh (spoilers), not really a Rubyist, so I tend to miss that kind of convention. I'll address that in this pass.